dev-docs/e2e/bare-metal-runner.md

```diff
@@ -1,5 +1,221 @@
+# How to add a bare-metal instance to the CI
+
+## Packets and updates (SNP)
+
+AMD SEV-SNP is supported by kernels newer than 6.11 (see https://www.phoronix.com/news/Linux-6.11-KVM).
+While we want to check occasionally if the latest version breaks
+Contrast, we want to use the latest LTS version.
```

Comment:
Why is that?

Reply:
I thought that we didn't have a guide on what version to use so I just made a recommendation based on my opinion. Though I think we should give _some_ recommendation here so that the dev executing the steps has to think as little as possible.